### PR TITLE
[Quickfix] Admin panel use 'form-control' class for selects

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by AugierLe42e** [comment]
Ready to merge.

File: app/views/admins/stats.html.haml [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
       .pull-right
         = form_tag('/admins/stats', :method => 'get', class: 'form-inline') do
-          %select{:name => 'range'}
+          %select.form-control{:name => 'range'}
```
Comment (jhass) [inline app/views/admins/stats.html.haml]: `name: "range"`

**Comment by AugierLe42e** [comment]
Done.

File: app/views/admins/stats.html.haml (hunk 1 above)
Comment (svbergerem) [inline app/views/admins/stats.html.haml]: ping

Comment (AugierLe42e) [inline app/views/admins/stats.html.haml]: I think you pinged an outdated diff :p

Comment (svbergerem) [inline app/views/admins/stats.html.haml]: You only changed to single quotes to double quotes but didn't change the hash syntax so the comment is still valid.

Comment (AugierLe42e) [inline app/views/admins/stats.html.haml]: That's right.

**Comment by svbergerem** [comment]
Thanks!

**Pull request merged**